<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigator Database Maintenance</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QTabWidget" name="dialog_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="fingerprints">
      <attribute name="title">
       <string>Access Point Fingerprints</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
         <item>
          <layout class="QHBoxLayout" name="floor_graphic_layout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Zoom:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="fp_zoom_slider">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="pageStep">
              <number>2</number>
             </property>
             <property name="value">
              <number>300</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Floor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fp_floor_combobox">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fp_new_capture_push_button">
             <property name="text">
              <string>New Capture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Show all locations</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="fp_new_capture_groupbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>New Capture Location:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X Pos:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="fp_x_pos_label">
                <property name="text">
                 <string>-99</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y Pos:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="fp_y_pos_label">
                <property name="text">
                 <string>-99</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Loc ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fp_loc_id_line_edit">
                <property name="text">
                 <string>1K999</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="fp_airdump_button">
                <property name="text">
                 <string>airodump it!</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="fp_insert_only_button">
                <property name="text">
                 <string>Insert Only</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="fp_cancel_button">
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Existing Capture Locations:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableWidget" name="locations_table">
              <column>
               <property name="text">
                <string>Loc ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>X Pos</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y Pos</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="measurements_table">
              <column>
               <property name="text">
                <string>HW Addr</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>SSID</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="fp_delete_capture_button">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Delete Capture</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="fp_redo_capture_button">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Redo Capture</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="assets">
      <attribute name="title">
       <string>Building Assets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0">
           <item>
            <layout class="QHBoxLayout" name="floor_graphic_layout_2"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Zoom:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="as_zoom_slider">
               <property name="minimum">
                <number>100</number>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="pageStep">
                <number>2</number>
               </property>
               <property name="value">
                <number>300</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Floor:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="as_new_capture_button">
               <property name="text">
                <string>Capture Asset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="as_cancel_button">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>X Pos:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="x_pos_label_2">
                 <property name="text">
                  <string>-99</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Y Pos:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="y_pos_label_2">
                 <property name="text">
                  <string>-99</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Office:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Room Number:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Owner's Name:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit">
                <property name="text">
                 <string>Last, First</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Owner's Role</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="comboBox_3">
                <item>
                 <property name="text">
                  <string>professor</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>researcher</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>other</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="pushButton_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Create Office</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Office Mate:</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Room Number:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBox_5"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Additional Owner's Name:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_3">
                  <property name="text">
                   <string>Last, First</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Additional Owner's Role:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox_4">
                  <item>
                   <property name="text">
                    <string>professor</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>researcher</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>other</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Add Office Mate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="specialized_groupbox">
         <property name="title">
          <string>Specialized Room or Lab:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Room Number:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Description:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add Room</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fp_zoom_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainDialog</receiver>
   <slot>fp_update_floor_scale(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fp_new_capture_push_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>fp_start_new_capture_mode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fp_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>fp_cancel_new_capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>941</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>1008</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fp_airdump_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>fp_perform_ap_capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>1012</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fp_insert_only_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>fp_insert_loc_id()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>1011</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>as_zoom_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainDialog</receiver>
   <slot>as_update_floor_scale(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fp_update_floor_scale(int)</slot>
  <slot>fp_start_new_capture_mode()</slot>
  <slot>fp_perform_ap_capture()</slot>
  <slot>fp_cancel_new_capture()</slot>
  <slot>fp_insert_loc_id()</slot>
  <slot>as_update_floor_scale(int)</slot>
 </slots>
</ui>
